skipIterator.py

```diff
@@ -0,0 +1,47 @@
+#Difficulties: How to check hasNext method in normal native tierator in python? 
```

Comment:
Python Iterators do not have a hasNext method, you can refer to this for more information.
https://docs.python.org/3/library/stdtypes.html#iterator-types